src/modules/0.0.14/aws_ec2/sql/create_custom_constraints.sql

```diff
@@ -30,11 +30,11 @@ $$;
 ALTER TABLE instance ADD CONSTRAINT check_role_ec2 CHECK (role_name is NULL OR (role_name is NOT NULL AND check_role_ec2(role_name)));
 
 -- Create EC2 instance and EBS volume must be in the same availability zone
-create or replace function check_instance_ebs_availability_zone(_instance_id integer, _ebs_availability_zone general_purpose_volume_availability_zone_enum) returns boolean
+create or replace function check_instance_ebs_availability_zone(_instance_id integer, _ebs_availability_zone character varying) returns boolean
```

Comment:
hmmm why this change?

Reply:
The availability zone is no longer an enum, it's a table that gets joined in now.

Reply:
ohh!